From: =?utf-8?q?David_Pr=C3=A9vot?= <taffit@debian.org>
Date: Tue, 20 May 2014 14:56:49 -0400
Subject: Drop Composer autoLoader calls

Work around the lack of proper autoload.php from Composer by using a
static ClassLoader made by PHPAB.

Forwarded: not-needed
---
 tests/bootstrap.php | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tests/bootstrap.php b/tests/bootstrap.php
index e74f387..7da7881 100644
--- a/tests/bootstrap.php
+++ b/tests/bootstrap.php
@@ -4,12 +4,6 @@ set_include_path(__DIR__ . '/../lib/' . PATH_SEPARATOR . __DIR__ . PATH_SEPARATO
 
 $autoLoader = include __DIR__ . '/../vendor/autoload.php';
 
-// SabreDAV tests auto loading
-$autoLoader->add('Sabre\\', __DIR__);
-// VObject tests auto loadiong
-$autoLoader->addPsr4('Sabre\\VObject\\',__DIR__ . '/../vendor/sabre/vobject/tests/VObject');
-
-
 date_default_timezone_set('UTC');
 
 $config = [
